packages/utils/src/jsonabc.cjs

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2018-2021 BOTLabs GmbH.
+ * Copyright (c) 2018-2022, BOTLabs GmbH.
```

Comment:
Unrelated to this PR but why are we claiming copyright over JSON ABC code?

Reply:
oh yeah, that file should be ignored from the linter